user_guide.md

```diff
@@ -447,6 +447,11 @@ command and flag definitions are needed.
 Help is just a command like any other. There is no special logic or behavior
 around it. In fact, you can provide your own if you want.
 
+### Grouping commands in help
+
+Cobra supports grouping of available commands. Groups can either be explicitly defined by `AddGroup` and set by
+the `Group` element of a subcommand. If Groups are not explicitly defined they are implicitly defined.
```

Comment:
> If Groups are not explicitly defined they are implicitly defined.

I think this is a bit confusing. Instead, it seems that the commands that don't have a group get bundled into the "Available commands" which is the current default behavior. When I originally read this, I wasn't sure what "implicitly defined" meant. Can we simplify this chunk?

Reply:
I agree we maybe should remove the ability to implicitly define groups. I'll remove this passage here.

Reply:
is changed.